src/components/avatar/AvatarContent.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon, {types as ICON_TYPE, colors as ICON_COLOR} from '../icons/Icon';
```

Comment:
it's kind of weird / tricky to pass this around.
I mean, it's a part of icon module, but we're exporting it in two places (icon/avatar-content) with different identifiers. @aju @kdzwinel wdyt?